src/fswAlgorithms/imageProcessing/opticalFlow/opticalFlow.cpp

```diff
@@ -152,12 +152,20 @@ void OpticalFlow::UpdateState(uint64_t CurrentSimNanos)
                                 false,
                                 0.04);
 
-        v3Copy(navAttBuffer.sigma_BN, this->firstSpacecraftAttitude);
-        v3Copy(ephemMsgBuffer.sigma_BN, this->firstTargetEphemAttitude);
-        this->firstTimeTag = this->sensorTimeTag;
-        this->firstFeatures = this->secondFeatures;
-        this->firstImagePresent = true;
-        this->secondImagePresent = false;
+        if (this->firstFeatures.size() > 0) {
```

Comment:
:large_orange_diamond: Probably better to use `if (this->firstFeatures.empty())` instead

Reply:
Done